### PR TITLE
Modular questions on front-end

### PR DESCRIPTION
**Pull request by amirrr**
## Describe your changes

1. Removed console.logs from production
2. Questions now load from JSON
3. Manifest.js only serves two treatments in two design spaces with only the cleaned statements

### CODE REVIEW

File: client/src/components/Enter.tsx
Comment (amirrr) [inline client/src/components/Enter.tsx:1]: Now after login user redirect to dashboard

File: server/survey/manifest.js
Comment (amirrr) [inline server/survey/manifest.js:1]: Only do 2 treaments

Comment (markwhiting) [reply server/survey/manifest.js:1]: This looks like it’s hard coded for Duncan?

File: server/survey/treatments/design-point.ts
Comment (markwhiting) [inline server/survey/treatments/design-point.ts:1]: We can probably do this as weighted exploration over an array of constant design points that we modify over time. So when we add more they become the focal ones.

File: client/src/components/QuestionOne.tsx
Comment (markwhiting) [inline client/src/components/QuestionOne.tsx:1]: Not sure how this is used. Is it needed if we are using the file driven method?

**Review by markwhiting — CHANGES_REQUESTED** [review]
Overall looks good but a few questions.

File: client/src/components/MultiStepForm.tsx [hunk 1]
```diff
@@ -60,27 +53,33 @@ function MultiStepForm(props) {
       if (!props.steps[currentStepIndex].answereSaved) {
         Backend.post("/answers", {
           statementId: props.steps[currentStepIndex].id,
-          I_agree: props.steps[currentStepIndex].answers[0].slice(-1),
+          I_agree:
+            props.steps[currentStepIndex].answers[0].split("-")[1] === "Yes"
+              ? 1
+              : 0,
           I_agree_reason:
```
Comment (markwhiting) [inline client/src/components/MultiStepForm.tsx:60]: I'll create an issue for removing hard coding of variable names around this.

Comment (markwhiting) [reply client/src/components/MultiStepForm.tsx:60]: (as in, we need not fix it now, but should in a next cycle)

**Review by markwhiting — APPROVED** [review]
Made some new issues but I think this looks good to go for now!

**Pull request merged**